cli/flox/src/commands/services/mod.rs

```diff
@@ -6,35 +6,42 @@ use flox_rust_sdk::providers::services::{ProcessState, ProcessStates, ServiceErr
 use tracing::instrument;
 
 use super::{ConcreteEnvironment, EnvironmentSelect};
+use crate::config::Config;
 
 mod logs;
+mod start;
 mod status;
 mod stop;
 
 /// Services Commands.
 #[derive(Debug, Clone, Bpaf)]
 pub enum ServicesCommands {
+    /// Ensure a service or services are running
+    #[bpaf(command, footer("Run 'man flox-services-start' for more details."))]
```

Comment:
I didn't add the man page yet, but I figure we're more likely to forget to add this later